### ISSUE TITLE
`index.html` and `bundle.js` are big

### ISSUE THREAD

**Issue by clehner** [dominictarr/patchbay#53]
The built files `index.html` and `bundle.js` are big (2mb each, 500k gz'd). It makes cloning and fetching slower. Do these files need to be checked in to git, or can be they be added in a prepublish script or manual step?

**Comment by dominictarr** [comment]
yeah... actually we should just stop using gh-pages, and use https://github.com/dominictarr/web-bootloader instead.

**Comment by dominictarr** [comment]
I'll put them in my .gitignore

**Comment by dominictarr** [comment]
fixed in b52121dfc8bdd03d92c43ebc8a4c024c940b3252

**Comment by mixmix** [comment]
@dominictarr if this is fixed, shall we close this ?
